packages/migration/src/migrations/v0011_pageAdvancement.ts

```diff
@@ -45,22 +45,15 @@ export type UnionElementV11 =
 
 export interface StoryV11 extends Omit<StoryV10, 'pages'> {
   pages: PageV11[];
-  autoAdvance: boolean;
-  defaultPageDuration: number;
+  autoAdvance?: boolean;
+  defaultPageDuration?: number;
 }
 export interface PageV11 extends Omit<PageV10, 'elements'> {
   elements: UnionElementV11[];
 }
 
 function pageAdvancement(story: StoryV10): StoryV11 {
```

Comment:
So this does nothing now?

As crazy is this sounds, I think it might be cleaner to add a new migration to remove default values.